src/components/TagContainer.ts

```
@@ -0,0 +1,277 @@
+                mxObject.addReference(this.referenceAttribute, object.getGuid());
+                this.saveTagData(mxObject);
+                this.setState({ tagList });
+
+                return;
+            }
+        }
+        mx.data.create({
+            callback: object => {
+                object.set(this.props.tagAttribute, newTag);
+                mx.data.commit({
+                    callback: () => {
+                        mxObject.addReference(this.referenceAttribute, object.getGuid());
+                        this.saveTagData(mxObject);
+                        if (afterCreateMicroflow || onChangeMicroflow) {
+                            this.executeAction(mxObject, afterCreateMicroflow);
+                            this.executeAction(mxObject, onChangeMicroflow);
+                        }
+                    },
+                    error: error => window.mx.ui.error("Error occurred attempting to commit: " + error.message),
+                    mxobj: object
+                });
+            },
+            entity: this.tagEntity,
+            error: error => window.mx.ui.error(`Error creating tag object ${this.tagEntity}, ${error.message}`)
+        });
+
+        tagList.push(newTag);
+        this.setState({ tagList });
+    }
+
+    private removeTag(value: string) {
+        if (value.trim() !== "") {
+            const { onChangeMicroflow, mxObject } = this.props;
+            const tagIndex = this.state.tagList.indexOf(value);
+            const tagList = this.state.tagList.slice();
+
+            mx.data.get({
+                callback: (object) => {
+                    if (onChangeMicroflow) {
+                        this.executeAction(mxObject, onChangeMicroflow);
+                    }
+                    mxObject.removeReferences(this.referenceAttribute, object[0].getGuid() as any);
+                    this.saveTagData(mxObject);
+                },
+                error: error => `${error.message}`,
```

Ohohh.. this error is never logged please use `window.mx.ui.error`
Never hide a real error that you can not recover from. 
And if you do no want to show the user, at lease log in in the console.